apis/apps/v1alpha1/cluster_types.go

```diff
@@ -339,6 +339,14 @@ type ClusterComponentSpec struct {
 	// +kubebuilder:default=1
 	Replicas int32 `json:"replicas"`
 
+	// Minimum number of seconds for which a newly created pod should be ready
+	// without any of its container crashing for it to be considered available.
+	// Defaults to 0 (pod will be considered available as soon as it is ready)
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:default=0
+	// +optional
+	MinReadySeconds int32 `json:"minReadySeconds,omitempty"`
```

Comment:
this MinReadySeconds filed should not be defined within the cluster and component APIs; it should be considered for defining in the ComponentDefinition or in the RSMSpec of ClusterDefinition.